### DISCUSSION TITLE
Formula Add Numbers

### DISCUSSION THREAD

**Discussion by Metalnuss** [RafaelGB/obsidian-db-folder#610] [Q&A]
Hello :D

Sorry if this is a super simple question. I am new to obsidian and also not good with coding.

I just need a formula that adds numbers in other columns.
I divided my Pomodoros in different categories and now just want to automatically add them to get the total amount of Pomodoros.

Thank you!

**Comment by RafaelGB** [comment]
Hello @Metalnuss , thanks for using the plugin.

The formulas were thought to be used per row. 
If you have multiple columns with the pomodoros you can generate a formula

`${row.pomodoroId1+row,pomodoroId2}`…

But if you want to add different row values, we need to implement a footer formula (not avaliable yet)

**Comment by Metalnuss** [comment]
Hello again!
Thank you for the quick answer. And also for providing this super helpful plugin!!

Actually I tried something similar but crashed the whole database with it. (Found out that my line breaks in the Formula were the real issue)

At least I was not that far off. I tried:
`${row.pomodoroId1}+${row.pomodoroId2}`
Which gives what It says... "1+2" (example) and not "3"

(In case this may be helpful for other newbies trying out a bunch of stuff but failing anyways)
